joomla/joomla-cms
# [6.0] Integer custom field to be numerical input instead of a drop-down

Pull Request for Issue #42579

### Summary of Changes
So far the Integer custom field is a drop-down list. 
Though it is crashing the browser when the list becomes too long. Moreover it is really hard to use, after a certain length.

With the PR the Integer custom field is an [input of type number](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number).  Though, it still follows the min, max and step settings, that existed.

### Testing Instructions
Try setting up a custom field of type "Integer" for an article.
Within the article you will see an input, instead of a drop-down,

### Actual result BEFORE applying this Pull Request
Drop-down list.

### Expected result AFTER applying this Pull Request
Input of type number

### Link to documentations
DKN
